user/applications/tinker/src/burnin_test.cpp

```diff
@@ -459,5 +473,147 @@ bool BurninTest::testCpuLoad() {
 	return true;
 }
 
+#if PLATFORM_ID == PLATFORM_MSOM
+
+static bool turnModemOn() {
+	Cellular.on();
+	waitFor(Cellular.isOn, 30000);
+
+	if (!Cellular.isOn()) {
+		strcpy(BurninErrorMessage, "Cell modem failed to turn on after 30s");
+		return false;
+	}
+	return true;
+}
+
+bool BurninTest::testCellularModem() {
+	if (!turnModemOn()) {
+		return false;
+	}
+	
+    CellularDevice device = {};
+    device.size = sizeof(device);
+    if (cellular_device_info(&device, NULL)) {
+		strcpy(BurninErrorMessage, "Failed to get cellular info");
+    	return false;
+    }
+    
+    Log.info("Cell modem ICCID: %s IMEI: %s FW: %s", device.iccid, device.imei, device.radiofw);
+
+	Cellular.off();
+	waitFor(Cellular.isOff, 30000);
+	return true;
+}
+
+static int callbackGPSGGA(int type, const char* buf, int len, bool* gnssLocked) {
+	// EXAMPLE:
+	// $<TalkerID>GGA,<UTC>,<Lat>,<N/S>,<Lon>,<E/W>,<Quality>,<NumSatUsed>,<HDOP>,<Alt>,M,<Sep>,M,<DiffAge>,<DiffStation>*<Checksum><CR><LF>
+	// +QGPSGNMEA: $GPGGA,213918.00,3804.405282,N,12209.922544,W,1,05,3.0,145.9,M,-26.4,M,,*5E
+
+    //Log.info("%s", buf);
+
+	*gnssLocked = false;
+
+	const int MAX_GPGGA_STR_LEN = 256;
+    char gpggaSentence[MAX_GPGGA_STR_LEN] = {};
```

Comment:
Id like to fix the mix of tabs/space (change all tabs to 4 spaces) in all the fqc/burnin files. 
Should I just do a whitespace/formatting only commit or will that make this too difficult to review?

Reply:
I think we can hide whitespace changes.  Go ahead and even squash commits if you want.

Reply:
We can hide whitespace changes so please do make a commit for whitespace changes, and also no need to squash it together.